zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/BackgroundTaskManager.java

```diff
@@ -101,7 +106,7 @@ public static Archiver create(
   }
 
   private static ProcessInstancesArchiverJob createProcessInstanceJob(
-      final CamundaExporterMetrics metrics,
+      final io.camunda.exporter.metrics.CamundaExporterMetrics metrics,
```

Comment:
The prefix can be removed it still points to the same place. This probably happened automatically 🤔